vllm/lora/lora.py

```diff
@@ -108,17 +108,20 @@ def __init__(
             lora_alpha=0,
             lora_a=lora_a,
             lora_b=lora_b,
-            scaling=scaling,
+            scaling=scaling,  # type: ignore
```

Comment:
Looks like `scaling` is `Optional[List[Optional[float]]]`?

Reply:
the thing is the parent class scaling is just Optional[float].

Talked with Antoni and he suggested to just remove super class init here

Reply:
Hmm tried removing super().__init__ here, but mypy still cannot handle it because self.scaling has different types.

I will probably just ignore for now. The logic itself is technically correct